experimental/traffic-portal/src/app/utils/index.spec.ts

```diff
@@ -99,7 +99,7 @@ describe("Typing utility functions", () => {
 	it("should check homogenous array types correctly", () => {
 		const a = {};
 		expect(isArray(a)).toBeFalse();
-		const b = new Array();
+		const b: unknown[] = [];
```

Comment:
did the upgrade make this change as a migration?

Reply:
It was a linting rule from the recommended (which is now removed), but for the record I think this change makes sense.

Reply:
well it's technically incorrect. Before the change, b was `Array<never>`, and after it's `Array<unknown>`. The way it's being used that doesn't matter, but it does seem odd to me that they would add that "fixer".